packages/portal-proto/src/components/Table/VerticalTable.tsx

```diff
@@ -32,6 +32,39 @@ import { createKeyboardAccessibleFunction } from "@/utils/index";
 import { v4 as uuidv4 } from "uuid";
 import { useDeepCompareEffect } from "use-deep-compare";
 
+/**
+ * VerticalTable is a table component that displays data in a vertical format.
+ * @param columns - The columns to be displayed in the table.
+ * @param data - The data to be displayed in the table.
+ * @param footer - The footer to be displayed in the table.
+ * @param getRowCanExpand - A function that returns a boolean value indicating whether a row can be expanded.
+ * @param expandableColumnIds - The column ids that can be expanded.
+ * @param setRowSelection - A function that sets the row selection.
+ * @param rowSelection - The row selection.
+ * @param enableRowSelection  - A boolean value indicating whether row selection is enabled.
+ * @param status  - The status of the data.
+ * @param tableTitle  - The title of the table.
+ * @param additionalControls  - Additional controls to be displayed in the table.
+ * @param search  - The search options for the table.
+ * @param showControls  - A boolean value indicating whether the controls should be displayed.
+ * @param handleChange  - A function that handles the change.
+ * @param pagination  - The pagination options for the table.
+ * @param disablePageSize - A boolean value indicating whether the page size should be disabled.
+ * @param renderSubComponent - A function that renders the sub component.
+ * @param columnVisibility  - The column visibility.
+ * @param setColumnVisibility - A function that sets the column visibility.
+ * @param columnOrder - The column order.
+ * @param setColumnOrder  - A function that sets the column order.
+ * @param columnSorting - The column sorting.
```

Comment:
`none` - no sorting
 `enable` - sorts data in table
`manual` - uses handleChange callback to enable manual sorting

Reply:
Added